doc/advanced_setup.rst

```diff
@@ -106,6 +106,19 @@ report them on the MNE mailing list or on github to get assistance.
 
 .. _inside_martinos:
 
+Installing Mayavi
+^^^^^^^^^^^^^^^^^
+
+In our experience, the easiest way to install `mayavi` is using Anaconda. Just do::
+
+    conda install mayavi
```

Comment:
if you use Anaconda. You can point to:

http://docs.enthought.com/mayavi/mayavi/installation.html